parquet/src/main/java/org/apache/iceberg/parquet/ParquetDictionaryRowGroupFilter.java

```diff
@@ -463,6 +464,11 @@ private <T> Set<T> dict(int id, Comparator<T> comparator) {
 
       return dictSet;
     }
+
+    @Override
+    public <T> Boolean handleNonReference(Bound<T> term) {
```

Comment:
Same for parquet too. The reason is that we support push-down transform functions now. However, the file format can't handle such filters.